app/views/teaching_events/about_git_events.html.erb

```diff
@@ -55,7 +55,7 @@
             </ul>
           <% end %>
           <div>
-            <h2 class="heading-s">Can't find what you're looking for?</h2>
+            <h4 class="heading-s">Can't find what you're looking for?</h2>
```

Comment:
I don't think this should be an h4 @garethwilliamsdesign @MaxineCl @caitlin-reidy? As an h4, it is nesting under the last event listed at the moment (which in this environment is the National one)

Shouldn't be just be an h3 - so under `Events in your region` but on the same level as each event listing?